lib/ansible/modules/debconf.py

```diff
@@ -134,21 +134,15 @@ def get_password_value(module, pkg, question, vtype):
     cmd = [getsel]
     rc, out, err = module.run_command(cmd)
     if rc != 0:
-        module.fail_json(msg="Failed to get the value '%s' from '%s'" % (question, pkg))
+        module.fail_json(msg=f"Failed to get the value '{question}' from '{pkg}': {err}")
 
-    desired_line = None
     for line in out.split("\n"):
         if line.startswith(pkg):
-            desired_line = line
-            break
-
-    if not desired_line:
-        module.fail_json(msg="Failed to find the value '%s' from '%s'" % (question, pkg))
-
-    (dpkg, dquestion, dvtype, dvalue) = desired_line.split()
-    if dquestion == question and dvtype == vtype:
-        return dvalue
-    return ''
+            (dpkg, dquestion, dvtype, *dvalue) = line.split('\t')
+            if dquestion == question and dvtype == vtype:
+                if len(dvalue) >= 1:
+                    return dvalue[0]
```

Comment:
shouldn't we return the whole value? isn't the only case when you get more than one when the type is a list?